run-container/README.md

```diff
@@ -0,0 +1,116 @@
+# How to run the container for the Single cell RNA Seq Snap workflow (ScRNASeqSnap)
+
+We have generated Dockerfile and Definition file that contain all tools, packages, and dependencies necessary to run the code and analyses modules in the `sc-rna-seq-snap` repository. These are customized for `Rstudio/R v4.4.0` and `Seurat v4.4.0`.
+
+
+## To use the container in an R interactive session on HPC:
+
+From an interactive node on HPC, the user can open their session. Please modify memory and resources as needed for the analysis module to run.
+```
+bsub -P hpcf_interactive -J hpcf_interactive -n 1 -q standard -R "rusage[mem=4G]" -Is "bash"
+```
+
+## Load specific version of Singularity
+
+Please note that a version of Singularity is installed by default on all the cluster nodes at St Jude HPC. 
+Otherwise the user needs to ensure and load Singularity module by running:
+```
+module load singularity/4.1.1
+```
+
+
+1. Clone the repository
+```
+git clone https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap.git
+```
+
+2. Pull the singularity container
+```
+singularity pull docker://wabuala/rstudio_4.4.0_seurat_4.4.0:latest
+```
+
+
+3. Start the singularity container
+
+a. To run from the terminal
+```
+bash run-terminal.sh
+```
+
+Then you may navigate to your module of interest, `./sc-rna-seq-snap/analyses/<module_of_interest>`. For example:
+```
+cd ./sc-rna-seq-snap/analyses/upstream-analysis
+Rscript -e "rmarkdown::render(‘01A_run_seurat_qc.Rmd', clean = TRUE)"
+```
+
+
+b. To run from Rstudio
+```
+bash run-rstudio.sh
+```
+
+The `run-rstudio.sh` is running at `IP_ADDR:PORT`. When RStudio launches, please click "Session" -> "Restart R".
```

Comment:
Maybe consider adding a note here that users may need to turn off any ad-blockers they use to load the IP_ADDR:PORT Rstudio properly (I had to turn mine off to get it to load the page without issues that seemed to be related to cookies).

Reply:
Again, as noted above, the container appears to have the incorrect version BiocManager being loaded

Reply:
> Maybe consider adding a note here that users may need to turn off any ad-blockers they use to load the IP_ADDR:PORT Rstudio properly (I had to turn mine off to get it to load the page without issues that seemed to be related to cookies).

Fixed here 060ef4c9cb78fa3ee898f9d0af88a5305a7e2cea.

Reply:
@sharonfreshour Please note these updates in the `README.md` file: 96f56124fb90d00192d50119c839aa7b45e00317